vendor/bat-native-ledger/src/bat/ledger/internal/database/database_migration.cc

```diff
@@ -64,107 +59,84 @@ void DatabaseMigration::Start(
   const int start_version = table_version + 1;
   auto transaction = ledger::DBTransaction::New();
   int migrated_version = table_version;
-  const int target_version = GetCurrentVersion();
+  const int target_version = braveledger_database::GetCurrentVersion();
 
   if (target_version == table_version) {
     callback(ledger::Result::LEDGER_OK);
     return;
   }
 
-  for (auto i = start_version; i <= target_version; i++) {
-    if (!Migrate(transaction.get(), i)) {
-      BLOG(0, "DB: Error with MigrateV" << (i - 1) << "toV" << i);
-      break;
-    }
+  std::vector<std::string> mappings {
+    "",
```

Comment:
Just a placeholder so the numbering is more natural?

Reply:
@emerick all good on this one?

Reply:
Yup!